deployment/build_and_push.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+VERSION=$(git rev-parse --short HEAD)
+ECR_HOST="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com"
+ECR_URL="${ECR_HOST}/peerbook"
+
+if [ -z "${AWS_REGION+unset}" ]; then
+    echo "AWS_REGION must be set to push the image"
+    exit 1
+else
+    echo "Login into ECR"
+    aws ecr get-login-password --region "${AWS_REGION}" | docker login --username AWS --password-stdin "${ECR_HOST}"
```

Comment:
Are you sure if this fails, the job exits here?

Reply:
yes